src/pattern-library/components/Library.tsx

```diff
@@ -246,7 +242,7 @@ function Demo({
       <div className="p-2 unstyled-text">
         {visibleTab === 'demo' && (
           <div
-            className="w-full p-8 rounded-md border border-stone-300 bg-slate-50 rounded-md"
+            className="w-full p-8 rounded-md border border-stone-300 bg-stone-50 rounded-md"
```

Comment:
Updated demo background color to harmonize better with updated page background colors (warm vs. cool).